src/index.ts

```diff
@@ -14,64 +15,73 @@ export class MyClass {
   );
 
   public start(): void {
-    SignSettingsInstance.canvasSections.forEach((canvasSection) => {
-      this.myPainter
-        .getCanvas()
-        .addCanvasSection(
-          new Board.CanvasSection(
-            canvasSection.name,
-            canvasSection.x,
-            canvasSection.y,
-            canvasSection.z,
-            canvasSection.width,
-            canvasSection.height,
-            canvasSection.representation,
-            canvasSection.overflow
-          )
-        );
-    });
+    // SignSettingsInstance.canvasSections.forEach((canvasSection) => {
```

Comment:
Any way to run this on start, (may have many `SignSettingsInstance.canvasSections`